src/app/application.ts

```diff
@@ -328,22 +328,30 @@ export namespace Application {
     let processResources = () => {
         logger.info('Copy main resources');
         fs.copy(path.resolve(__dirname + '/../src/resources/'), path.resolve(process.cwd() + path.sep + defaultFolder), function (err) {
-            if (err) {
-                logger.error('Error during resources copy');
-            } else {
-                if (program.extStyle) {
-                    fs.copy(path.resolve(process.cwd() + path.sep + program.extStyle), path.resolve(process.cwd() + path.sep + defaultFolder + '/styles/bootstrap.min.css'), function (err) {
+            if(err) {
+                logger.error('Error during external styling file copy ', err);
```

Comment:
is still there. It has been moved on line 332. the logic is:
1) I copy everything
2) I check if extTheme is present. If yes I remove the copied styles folder and I copy the newOne